src/couch/src/couch_debug.erl

```diff
@@ -50,151 +50,181 @@ help() ->
 
 -spec help(Function :: atom()) -> ok.
 help(opened_files) ->
-    io:format("
-    opened_files()
-    --------------
-
-    Returns list of currently opened files
-    It iterates through `erlang:ports` and filters out all ports which are not efile.
-    It uses `process_info(Pid, dictionary)` to get info about couch_file properties.
-    ---
-    ", []);
+    io:format(
```

Comment:
This is probably a case where we should suppress reformatting by adding `%% erlfmt-ignore`.

```
-spec help(Function :: atom()) -> ok.
%% erlfmt-ignore
help(opened_files) ->
    ....
```

```

Reply:
Done in a875cba